ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,185 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        function restore-cluster-state() {
+          echo "Placing file /etc/rollback-test with contents A"
+          cat > /tmp/machineconfig.yaml <<'EOF'
+          apiVersion: machineconfiguration.openshift.io/v1
+          kind: MachineConfig
+          metadata:
+            labels:
+              machineconfiguration.openshift.io/role: master
+            name: 99-rollback-test
+          spec:
+            config:
+              ignition:
+                config: {}
+                security:
+                  tls: {}
+                timeouts: {}
+                version: 2.2.0
+              networkd: {}
+              passwd: {}
+              storage:
+                files:
+                - contents:
+                    source: data:,A
+                    verification: {}
+                  filesystem: root
+                  mode: 420
+                  path: /etc/rollback-test
+              systemd: {}
+        EOF
+          oc create -f /tmp/machineconfig.yaml
+
+          function wait_for_machineconfigpool_to_apply() {
+
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updating --timeout=5m && break; done
+            for i in $(seq 0 10); do oc wait machineconfigpool/master --for=condition=Updated --timeout=5m && break; sleep 30; done
+          }
+
+          wait_for_machineconfigpool_to_apply
+
+          echo "Setting up ssh bastion"
+          mkdir -p ~/.ssh || true
+          cp "${KUBE_SSH_KEY_PATH}" ~/.ssh/id_rsa
+          chmod 0600 ~/.ssh/id_rsa
+          echo "${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin" >> /etc/passwd
+          curl https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh | bash
+          BASTION_HOST=$(oc get service -n openshift-ssh-bastion ssh-bastion -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

Comment:
If we're going to run e2e tests after this, running the bastion host in this namespace could cause tests to fail: https://github.com/openshift/origin/blob/master/test/extended/operators/images.go#L81

Consider changing the namespace or removing the bastion host at the end of the tests

Reply:
This is resolved by removing `openshift-ssh-bastion` namespace on success.

The namespace cannot be easily changed, as its defined in `deploy.sh`

Reply:
That test failed on the last run:
```
fail [github.com/openshift/origin/test/extended/operators/images.go:112]: May  4 08:39:58.641: Pods found with invalid container images not present in release payload: openshift-ssh-bastion/ssh-bastion-f4d5bbcbd-vs6b4/ssh-bastion image=quay.io/eparis/ssh:latest
```
 but apparently the test suite can still pass with that test failing, so I don't think it is an issue.

Reply:
Weird, I see the project has been removed:
```
project.project.openshift.io "openshift-ssh-bastion" deleted
```
Perhaps pods didn't get scaled down - I'll fix that in new PR

Reply:
Implemented a proper removal of ssh bastion